src/backend_common.cc

```diff
@@ -1356,4 +1358,17 @@ BufferAsTypedString(
   return nullptr;  // success
 }
 
+std::string
+GetRequestId(TRITONBACKEND_Request* request)
+{
+  const char* request_id;
```

Comment:
Initialize `const char* request_id = nullptr` for consistency

Reply:
Fixed. Thanks.